lms/djangoapps/verify_student/tests/test_models.py

```diff
@@ -438,47 +437,3 @@ def test_active_at_datetime(self):
         user = UserFactory.create()
         verification = ManualVerification.objects.create(user=user)
         self.verification_active_at_datetime(verification)
-
-
-class VerificationDeadlineTest(CacheIsolationTestCase):
```

Comment:
Maybe worth keeping this to assert that there's always a single query?

Reply:
I'd rather just leave it out, honestly. If you feel strongly about it, I'll add it, but it was a single query before and that query was horrible. Switching it to do `n` queries would have actually probably been better for the vast majority of users.

The dashboard is definitely a mess of n+1 queries that we need to guard against, but I feel like we need some more basic changes in the design to guide people towards performant behavior.